controllers/providers/pg/applier.go

```diff
@@ -586,11 +586,21 @@ func (p *applier) genPGDataSourceSpec() (*crunchyv1beta1.DataSource, error) {
 	return pgDataSource, nil
 }
 
-func getPGRestoreOptions(dataSource everestv1alpha1.DataSource, backupBaseName string) ([]string, error) {
+func getPGRestoreOptions(
+	dataSource everestv1alpha1.DataSource,
+	backupStorage *everestv1alpha1.BackupStorage,
+	backupBaseName, repoName string) ([]string, error) {
 	options := []string{
 		"--set=" + backupBaseName,
 	}
 
+	if pointer.Get(backupStorage.Spec.ForcePathStyle) {
+		options = append(options, "--"+fmt.Sprintf(pgBackRestStorageForcePathTmpl, repoName)+"="+pgBackRestStoragePathStyle)
+	}
+	if !pointer.Get(backupStorage.Spec.VerifyTLS) {
+		options = append(options, "--no-"+fmt.Sprintf(pgBackRestStorageVerifyTmpl, repoName))
+	}
```

Comment:
I'm a bit confused why we need this at all. Should [this](https://github.com/percona/everest-operator/blob/939c65dc1771cef0b9ef2a645f173fa15f219809/controllers/providers/pg/applier.go#L1116-L1123) already take care of it? :thinking:

Reply:
That's what I thought at first, but it doesn't. The snippet you provided only takes care of the backups.
When you restore to a new cluster, those settings are lost. So you need to set them again

Reply:
This `getPGRestoreOptions` function is called only when we specify the `DataSource` field. Whereas in the snipped provided above, those are used only for backups and restoring to the same cluster. So I don't think there should be any conflicts/issues..

Reply:
BTW, I did a run through of the backup/restore scenario with these changes, and it worked as expected

Reply:
The thing that's concerning me is the hardcoded `repo1` (I know it was already like this).

Reply:
Okay so I just dug a bit deeper on this and I think everything is fine. Like you said this is completely isolated from the actual DB. The repos that we define here with the `handlePGDataSourceS3` have absolutely nothing to do with the ones defined outside the `DataSource` field.